skbuild/resources/cmake/FindPythonExtensions.cmake

```diff
@@ -317,6 +317,36 @@ if(NOT DEFINED PYTHON_EXTENSION_MODULE_SUFFIX)
   set(PYTHON_EXTENSION_MODULE_SUFFIX "${_item}")
 endif()
 
+function(_set_python_extension_symbol_visibility _target)
+  get_target_property(LANG ${_target} LINKER_LANGUAGE)
+  set(_linker_id "${CMAKE_${LANG}_COMPILER_ID}")
+
+  if(MSVC)
+    set_target_properties(${_target} PROPERTIES LINK_FLAGS 
+        "/EXPORT:PyInit_${_target}"
+    )
+  elseif("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU" AND
+         "${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU" AND
+         "${CMAKE_Fortran_COMPILER_ID}" STREQUAL "GNU")
```

Comment:
What I really want to say is that "this extensions will be linked with ld." If there's a better way to do that, please say so.